en/developer_guide/elasticsearch_indexation.rst

```diff
@@ -0,0 +1,137 @@
+
+Run the provisionning (``vagrant provision``) to install `Elasticsearch`_.
+
+Go on http://192.168.33.10:9200/ to check if it is running properly. You should see ``status: 200`` in
+the json response.
+
+Go on http://192.168.33.10:9200/_plugin/head/ to see the `plugin head`_ working.
+
+Elasticsearch and Open Orchestra
+--------------------------------
+
+To perform all the indexation in `Elasticsearch`_ an Open Orchestra bundle has been created. It is the
+`open-orchestra-elastica-bundle`_.
+
+Add in your ``composer.json`` file :
+
+.. code-block:: json
+
+    "open-orchestra/open-orchestra-elastica-bundle": "*"
+
+In the ``app/AppKernel.php`` file :
+
+.. code-block:: php
+
+    new OpenOrchestra\ElasticaBundle\OpenOrchestraElasticaBundle(),
+
+If your `Elasticsearch`_ is not listening on the ``127.0.0.1`` address, add the configuration in the
+``app/config/config.yml`` file :
+
+.. code-block:: yaml
+
+    open_orchestra_elastica:
+        host: 192.168.33.10
+
+After running the ``composer update`` command, you would see some new command appear when running
+``php app/console`` :
+
+.. code-block:: bash
+
+    $ php app/console | grep orchestra
+
+Those commands are :
+
+.. code-block:: bash
+
+    orchestra:elastica:index:create         Create an index in elastic search
+    orchestra:elastica:index:drop           Drop the index in elasticsearch
+    orchestra:elastica:populate             Populate the content index with the contents
+    orchestra:elastica:schema:create        Load the schema from the content types
+
+Creating the index
+~~~~~~~~~~~~~~~~~~
+
+The first time you install `Elasticsearch`_ you can use the command ``orchestra:elastica:index:create``
+to create the index.
+
+.. code-block:: bash
+
+    $ php app/console orchestra:elastica:index:create
+
+For more advanced users, you can directly go on `Elasticsearch`_ and create your index with the name ``content``.
+
+There is only an output if there is an error during the process.
+
+Creating the schema
+~~~~~~~~~~~~~~~~~~~
+
+Once your index is created, you should create the schema to help `Elasticsearch`_ store and retrieve your datas.
+
+The first time you are using `Elasticsearch`_ on an existing installation, you should use the command :
+
+.. code-block:: bash
+
+    $ php app/console orchestra:elastica:schema:create
+
+During the project lifetime, the schema will be automatically updated each time the ``ContentType`` are updated.
+
+There is only an output if there is an error during the process.
+
+Populating the index
+~~~~~~~~~~~~~~~~~~~~
+
+The first time you are using `Elasticsearch`_, you could populate the index with the existing datas, using
+the command :
+
+.. code-block:: bash
+
+    $ php app/console orchestra:elastica:index:create
+
+During the project lifetime, the indexed data will be automatically updated each time you publish a ``Content``.
+
```

Comment:
Is there a web graphic interface to monitor the indexed data ? If yes, it could be the location in the doc to indicate the adress.